.github/workflows/scripts/script.sh

```diff
@@ -20,6 +20,11 @@ export LANG=en_US.UTF-8
 sudo localectl status
 localectl status
 
+# Verbosity
+sed -i "/\[def/a verbosity = 2" ansible.cfg
```

Comment:
```
To see the results of individual retries, run the play with -vv.
```
https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met